post-scan-actions/aws-python-publish-malware-to-sns/template.yml

```
@@ -0,0 +1,90 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cloudone-filestorage-plugin-publish-malware-to-sns
+    Description: >-
+      According to the scan result from the scanner, publish the malware information
+      to the target SNS topic.
+    Author: Trend Micro Cloud One File Storage Security
+
+Parameters:
+  TargetSnsTopicName:
+    Type: String
+    Description: The name for the SNS topic to which malware information publish.
+  PublishMalwareLambdaName:
+    Type: String
+    Description: The name for the lambda function publishes the malware information to SNS topic.
+  PublishMalwarePolicyName:
+    Type: String
+    Description: The name for the policy includes sns and account authority.
+  PublishMalwareRoleName:
+    Type: String
+    Description: The name for the role to execute the lambda function.
+  ScanResultTopicARN:
+    Type: String
+    Description: The ARN of the scan result SNS topic in storage stack.
+Resources:
+  TargetSnsTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: !Ref TargetSnsTopicName
+      DisplayName: Malware notification
+  PublishMalwarePolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      ManagedPolicyName: !Ref PublishMalwarePolicyName
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - 'sns:TagResource'
+              - 'sns:Publish'
+              - 'account:GetAlternateContact'
```

```suggestion
              - sns:TagResource
              - sns:Publish
              - account:GetAlternateContact
```
And what's `sns:TagResource` for?

---

`sns:TagResource` is redundant, I will remove it in next commit.